### ISSUE TITLE
Add a filter for 'Regular Quests'

### ISSUE THREAD

**Issue by ctdgunner** [Zoinkwiz/quest-helper#1775]
Add a filter for 'Regular Quests'
There is one for 'Miniquest' and 'Achievement Diary' I'm looking for an option that hides those two and just shows the quest that give quest points. I want the cape after all ;) Not the end of the world lol I just feel if we have the other two we should have the 3rd and final for regular quest, right?

![image](https://github.com/user-attachments/assets/fb09842c-49a1-4241-9212-77b7700eed8d)

**Comment by ctdgunner** [comment]
Forgot to add. Quest Points Asc and Dsc do this exact filtering, BUT you do loose the "optimal ordering" if you care about that like I do.

**Comment by Zoinkwiz** [comment]
Hey, under the 'Filters' option there's a Quest option will that work?

**Comment by ctdgunner** [comment]
yes that does work i suppose i didnt notice because i usually leave that on 'show meets reqs' So doing show meets reqs and filtering to quest isnt possible like it would be mini's and diaries.

this is good enough for me now for sure but that little disconnect would have been more annoying a month or so ago when i had many more quest left.. i only have like 10 left for me
